freeipa/src/main/java/com/sequenceiq/freeipa/events/FreeipaUrlParser.java

```diff
@@ -0,0 +1,55 @@
+package com.sequenceiq.freeipa.events;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.structuredevent.event.CloudbreakEventService;
+import com.sequenceiq.cloudbreak.structuredevent.rest.urlparser.CDPRestUrlParser;
+
+@Component
+public class FreeipaUrlParser extends CDPRestUrlParser {
```

Comment:
shall we introduce other parsers for the other endpoints? like `KerberosMgmtV1Endpoint`, `KerberosConfigV1Endpoint`, `LdapConfigV1Endpoint`, `DnsV1Endpoint`?

Reply:
Good questions, I will ask

Reply:
Only Kerberos config and Ldap config added